src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,11 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Machine[] machines = {new Bulldozer(), new Excavator(), new Truck()};
+        for (Machine m : machines) {
```

Comment:
```suggestion
        for (Machine machine : machines) {
```
bad naming, please don't shorten to one letter, it's not a math problem :)
we spend more time reading code than actually writing it in this indsutry